packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -1269,3 +1274,312 @@ EmptyStateTable = () => {
   );
 }
 ```
+
+```js title=Editable-Rows isBeta
```

Comment:
Can you make this title sentence case please.
I noticed that some of the other Table example titles are not sentence case either.  Can you fix those while you are in that file?